util/fipstools/acvp/acvptool/subprocess/subprocess.go

```diff
@@ -107,16 +133,18 @@ func NewWithIO(cmd *exec.Cmd, in io.WriteCloser, out io.ReadCloser) *Subprocess
 		"hmacDRBG":          &drbg{"hmacDRBG", map[string]bool{"SHA-1": true, "SHA2-224": true, "SHA2-256": true, "SHA2-384": true, "SHA2-512": true}},
 		"KDF":               &kdfPrimitive{},
 		"KDA":               &hkdf{},
+		"TLS-v1.2":          &tlsKDF{},
+		"TLS-v1.3":          &tls13{},
 		"CMAC-AES":          &keyedMACPrimitive{"CMAC-AES"},
 		"RSA":               &rsa{},
 		"kdf-components":    &kdfComp{"kdf-components"},
-		"TLS-v1.2":          &kdfComp{"TLS-v1.2"},
```

Comment:
Looks like the acvp commit somehow stealth added support for TLS 1.2 and 1.3 for Google. The TLS 1.3 stuff is great, but we already support TLS 1.2 through some code in `kdf-components.go`.

I have a commit where I renamed their `tlskdf.go` to `kdf-components` because we wanted to support SSH KDFs (and for some reason NIST uses the same algorithm name in their ACVP specs for SSH and TLS KDFs). We also support TLS 1.0/1.1 in addition to TLS 1.2, whereas it looks like Google's implementation still only supports TLS 1.2.

All that just to say that we should probably continue using our implementation via `&kdfComp{"TLS-v1.2"}` instead of what Google has here.

Reply:
Agreed. This should be resolved with latest update.